commands/operator-sdk/cmd/new.go

```diff
@@ -94,6 +101,38 @@ func mustBeNewProject() {
 	}
 }
 
+func doScaffold() {
+	// create cmd/manager dir
+	fullProjectPath := filepath.Join(mustGetwd(), projectName)
+	cmdDir := filepath.Join(fullProjectPath, "cmd", "manager")
+	if err := os.MkdirAll(cmdDir, defaultDirFileMode); err != nil {
+		log.Fatalf("failed to create %v: %v", cmdDir, err)
+	}
+
+	// generate cmd/manager/main.go
+	cmdFilePath := filepath.Join(cmdDir, "main.go")
+	cmdgen := scaffold.NewCmdCodegen(&scaffold.CmdInput{ProjectPath: repoPath()})
+	buf := &bytes.Buffer{}
+	err := cmdgen.Render(buf)
+	if err != nil {
+		log.Fatalf("failed to render template: %v", err)
```

Comment:
We should make it clear what file the render failed for.

```Go
log.Fatalf("failed to render template for (%v): %v", cmdFilePath, err)
```

Reply:
sg